web/apps/playground/src/components/BottomPanel.tsx

```diff
@@ -0,0 +1,67 @@
+import type React from "react";
+import { forwardRef } from "react";
+import { useAtomValue } from "jotai";
+import { annotationAtom, sampleTaskAtom } from "../atoms/configAtoms";
+import { IconCollapseSmall, IconExpandSmall } from "@humansignal/icons";
+import { cnm } from "@humansignal/ui/utils/utils";
+
+export type BottomPanelRef = {
+  handleAnnotationUpdate: (annotation: any) => void;
+};
+
+interface BottomPanelProps {
+  isCollapsed: boolean;
+  setIsCollapsed: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const HEADER_HEIGHT = 33;
+
+export const BottomPanel = forwardRef<BottomPanelRef, BottomPanelProps>(({ isCollapsed, setIsCollapsed }, ref) => {
+  const currentAnnotation = useAtomValue(annotationAtom);
+  const sampleTask = useAtomValue(sampleTaskAtom);
+
+  return (
+    <div
+      className={cnm("flex flex-col transition-all duration-200 min-h-0 min-w-0 h-full", {
+        "border-t border-neutral-border": isCollapsed,
+      })}
+    >
+      {/* Header (always visible, 33px) */}
+      <div
+        className="relative h-[33px] flex flex-row items-center bg-neutral-surface select-none"
+        style={{ minHeight: HEADER_HEIGHT, maxHeight: HEADER_HEIGHT }}
```

Comment:
this should do the same thing with out inline styles but if we have the height set - why do we need min and max set too?
```suggestion
        className={`relative h-[${HEADER_HEIGHT}px] flex flex-row items-center bg-neutral-surface select-none min-h-[${HEADER_HEIGHT}px] max-h-[${HEADER_HEIGHT}px]`}
```

Reply:
yeah, if we need to have this height fixed:
- `h-[${HEADER_HEIGHT}px]`
- no min/max
- flex-shrink: 0